### PR TITLE
[IMP] fieldservice_account: Tests

### PR DESCRIPTION
**Pull request by brian10048**
Tests should test something

### CODE REVIEW

File: fieldservice_account/__manifest__.py [hunk 1]
```diff
@@ -15,6 +15,6 @@
         "views/fsm_order.xml",
     ],
     "license": "AGPL-3",
-    "development_status": "Beta",
+    "development_status": "Production/Stable",
```
Comment (ghost) [inline fieldservice_account/__manifest__.py:18]: Any reason to move this to Production/Stable?

Comment (max3903) [reply fieldservice_account/__manifest__.py:18]: @gbrito-erpopen This module is only needing tests to move to `Production/Stable". Makes sense to me. :+1:

Comment (ghost) [reply fieldservice_account/__manifest__.py:18]: As long as it has tests, the module is ready for Production/Stable, correct? clear

**Comment by brian10048** [comment]
Tests depend on #1004

File: fieldservice_account/__manifest__.py (hunk 1 above)
Comment (max3903) [reply fieldservice_account/__manifest__.py:18]: Correct.

**Review by bodedra — APPROVED** [review]
LG 👍

**Review by max3903 — APPROVED** [review]

**Pull request merged**